tests/UI/pages/FO/home/index.js

```diff
@@ -83,6 +88,40 @@ class Home extends FOBasePage {
     return this.elementVisible(page, this.homePageSection, 3000);
   }
 
+  /**
+   * Click on right/left arrow of the slider
+   * @param page {Page} Browser tab
+   * @param direction {string} Direction to click on
+   * @returns {Promise<void>}
+   */
+  async clickOnLeftOrRightArrow(page, direction) {
+    await page.click(this.carouselControlDirection(direction));
+  }
+
+  /**
+   * Is slider visible
+   * @param page {Page} Browser tab
+   * @param position {number} The slider position
+   * @returns {Promise<boolean>}
+   */
+  async isSliderVisible(page, position) {
+    await this.waitForVisibleSelector(page, this.carouselSliderId);
+
+    return this.elementVisible(page, `${this.carouselSliderInner}:nth-child(${position})`, 1000);
```

Comment:
Maybe use selectors for this (see other comment)

Reply:
Done thx :+1: